models/issue.go

```diff
@@ -68,6 +68,24 @@ func init() {
 	issueTasksDonePat = regexp.MustCompile(issueTasksDoneRegexpStr)
 }
 
+func (issue *Issue) totalTimes(e Engine) (string, error) {
+	times, err := GetTrackedTimes(FindTrackedTimesOptions{IssueID: issue.ID})
```

Comment:
It would be better to select total time from database already as single value

Reply:
This would mean a higher maintenance effort, wouldn't it?

Reply:
Just reuse `FindTrackedTimesOptions` and use `x.Where(options.ToCond()).Sum(&TrackedTime{}, "time")`